spec/requests/list_controller_spec.rb

```diff
@@ -1283,6 +1283,35 @@
       end
     end
 
+    it "should filter with tag_group option" do
+      topic_with_tag = Fabricate(:topic, tags: [tag])
+      topic2_with_tag = Fabricate(:topic, tags: [tag])
+      Fabricate(:tag_group, tags: [tag], name: "tag_group")
```

Comment:
Just for the clarity of the next person reading, I would name the tag group something other than `tag_group`.